wrench/README.md

```diff
@@ -9,17 +9,23 @@ Binary recordings can be generated by webrender and replayed with `wrench replay
 ```rust
 RendererOptions {
     ...
-    enable_recording: true,
+    recorder: Some(Box::new(BinaryRecorder::new("wr-frame.bin"))),
     ...
 }
 ```
 
-The recordings will be put in a folder named `record`.
+If you are working on gecko integration you can enable recording in `webrender_bindings/src/bindings.rs` by setting
```

Comment:
That's a weird documentation line for WR, given that there is no `webrender_bindings` path in this repo.